PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -76,7 +87,8 @@ def __is_valid_mnemonic(mnemonic: str) -> bool:
 def __is_valid_seed(seed: bytes) -> bool:
     """Check whether provided bytes represent a valid seed.
     """
-    pass
+    if not (type(seed) == bytes) or (len(seed) != SEED_LEN): return False
+    else: return True
```

Comment:
What about just
~~~python
return isinstance(seed, bytes) and len(seed) == SEED_LEN
~~~

Can length of seed (`SEED_LEN`) be reconfigured without breaking the BIP39 algorithm?

Reply:
What do you mean? The length of the seed was taken just 'cause of SHA512. I'm not sure how soon it'll be cracked.

Reply:
> What about just
Done, thanks.

Reply:
> What do you mean? The length of the seed was taken just 'cause of SHA512. I'm not sure how soon it'll be cracked.

My comment was more about whther SEED_LEN should be global variable or constant in this function. :)